src/scene/vcFolder.cpp

```diff
@@ -93,8 +93,24 @@ void vcFolder::AddToScene(vcState *pProgramState, vcRenderData *pRenderData)
 
     HandleNodeSelection(pProgramState, pNode);
 
+    if (pProgramState->pGotGeo == nullptr && pProgramState->getGeo && pNode->itemtype == vdkPNT_PointCloud && ((vcModel *)(pNode->pUserData))->m_pPreferredProjection != nullptr)
+    {
+      vcModel *pModel = (vcModel *)pNode->pUserData;
+      if (pProgramState->pGotGeo == nullptr)
+      {
+        pProgramState->getGeo = false;
+        pProgramState->pGotGeo = pModel;
+        vcProject_UseProjectionFromItem(pProgramState, pModel);
+      }
+    }
+
+    if (pProgramState->pGotGeo != nullptr)
```

Comment:
Why is this after the above block now? It used to be before?

Reply:
I was mistakenly thinking we needed to changeprojection on the source model, which previously might have happened after cycling through every scene item and finding the source model again before unsetting gotGeo